### ISSUE TITLE
Closing of other people's cases in the AH

### ISSUE THREAD

**Issue by Michael21107** [SuperPyroManiac/LSPDFR-Helper#37]
When using the `/close case` command, people can see and choose cases from others, despite not having perms to close them. When such a command is sent, the bot doesn't respond (''The application did not respond'').

**Comment by SuperPyroManiac** [comment]
So anyone can close anyone elses cases? As well as it does not reply? Sorry I have a hard time reproducing this on my end since in my testing its just me. Can you explain in more details and what steps are needed?

**Comment by Michael21107** [comment]
No, people can't close other's cases, just in the command all cases are shown.
